INSTALLATION.md

```diff
@@ -1,29 +1,128 @@
 # Installation
 
-The latest stable version of _taipy-gui_ is available through _pip_:
+There are different ways to install Taipy GUI, depending on how
+you plan to use it.
+
+If your goal is to look into the code, modify and improve it, go straight
+to the [source installation](#installing-for-development) section.
+
+Taipy GUI needs your system to have Python 3.8 or above installed.
+
+## Installing from PyPI
+
+The easiest way to install Taipy GUI is from the
+[Pypi software repository](https://pypi.org/project/taipy-gui/).
+
+Run the command:
+```console
+$ pip install taipy-gui
 ```
-pip install taipy-gui
+
+If you are running in a virtual environment, you will have to
+issue the command:
+```console
+$ pipenv install taipy-gui
```

Comment:
mention conda ?

Reply:
We're not ready yet for explaining a proper Conda installation.
That needs to be done soon enough though. Good point.